data/scenarios/02Tutorials/01-turn.yaml

```diff
@@ -36,21 +59,26 @@ robots:
       - compass
```

Comment:
Perhaps this tutorial could work without the compass. And that might be better because the next one does not necessarily have it installed.

Reply:
It could, but I would rather make sure every challenge had enough compasses so that beginners do not have to worry yet about using `north` vs `right`.

I am open to making a compass-free tutorial, but I did not intend it here. :slightly_smiling_face:

Reply:
I agree, they will figure it out easily enough later when they try a cardinal direction and it tells them they need a compass.